src/core/grid_based_algorithms/lb_particle_coupling.cpp

```diff
@@ -180,9 +219,12 @@ void add_swimmer_force(Particle &p) {
   }
 }
 #endif
+
 } // namespace
 
-void lb_lbcoupling_calc_particle_lattice_ia(bool couple_virtual) {
+void lb_lbcoupling_calc_particle_lattice_ia(
+    bool couple_virtual, const ParticleRange &local_particles,
+    const ParticleRange &more_particles) {
```

Comment:
what are ```more_particles``` supposed to be? Please add a docstring

Reply:
Well propose a better name. I'd prefer just calling the function twice for each range, but that is not possible because of the GPU stuff.

Reply:
okay different question: do you think that a dev in 3 years still knows what```more_particles``` are supposed to be without looking into the source code?